hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestDataUtil.java

```diff
@@ -111,6 +111,20 @@ public static void createKey(OzoneBucket bucket, String keyName,
         ReplicationType.RATIS, content);
   }
 
+  public static void createKey(OzoneBucket bucket, String keyName,
+                               byte[] content) throws IOException {
+    createKey(bucket, keyName, ReplicationFactor.ONE,
+        ReplicationType.RATIS, new String(content, UTF_8));
```

Comment:
`createKey(..., String)` should call `createKey(..., byte[])`, not the other way around.

Because `createKey(..., String)` writes bytes after all:

https://github.com/apache/ozone/blob/7b48861a9b44bc4c47a948f32582963153a8b388/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestDataUtil.java#L142